### DISCUSSION TITLE
feed/article/tSyMTdXlyZf06JrxPwid

### DISCUSSION THREAD

**Discussion by giscus** [wisdomekpotu/devhouse#23] [Comments]
# feed/article/tSyMTdXlyZf06JrxPwid

Web site created using create-react-app

https://www.devhouse.live/feed/article/tSyMTdXlyZf06JrxPwid

**Comment by wisdomekpotu** [comment]
# nice comment

**Reply by wisdomekpotu** [reply]
comment
